src/runtime-rs/config/configuration-cloud-hypervisor.toml.in

```diff
@@ -0,0 +1,350 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# XXX: WARNING: this file is auto-generated.
+# XXX:
+# XXX: Source file: "@CONFIG_CLH_IN@"
+# XXX: Project:
+# XXX:   Name: @PROJECT_NAME@
+# XXX:   Type: @PROJECT_TYPE@
+
+[hypervisor.cloud-hypervisor]
+path = "@CLHPATH@"
+kernel = "@KERNELPATH_CLH@"
+image = "@IMAGEPATH@"
+
+# rootfs filesystem type:
+#   - ext4 (default)
+#   - xfs
+#   - erofs
+rootfs_type=@DEFROOTFSTYPE@
+ 
+# Block storage driver to be used for the VM rootfs is backed
+# by a block device.
+vm_rootfs_driver = "@VMROOTFSDRIVER_CLH@"
```

Comment:
This is being set to `"virtio-blk-pci"` but it needs to be set to `"virtio-pmem"` by the build.

Reply:
I had tested this with virtio-blk-pci and it worked for me. I suppose the virtio-pmem requirement is for TDX?